hal/src/main/native/sim/CANAPI.cpp

```diff
@@ -147,6 +147,27 @@ void HAL_WriteCANPacketRepeating(HAL_CANHandle handle, const uint8_t* data,
   can->periodicSends[apiId] = repeatMs;
 }
 
+void HAL_WriteCANRTRFrame(HAL_CANHandle handle, int32_t length, int32_t apiId,
+                          int32_t* status) {
+  auto can = canHandles->Get(handle);
+  if (!can) {
+    *status = HAL_HANDLE_ERROR;
+    return;
+  }
+  auto id = CreateCANId(can.get(), apiId);
+  id |= HAL_CAN_IS_FRAME_REMOTE;
+  uint8_t data[8];
+  std::memset(data, 0, sizeof(data) / sizeof(uint8_t));
```

Comment:
Shouldn't this simply be `sizeof(data)` (it doesn't matter, because `sizeof(uint8_t)` is 1, but memset takes # of bytes, not # of elements.

Reply:
Oh, this is supposed to be `*`, I got it backwards. But yeah it can just be removed for our platforms.